horace_core/sqw/@DnDBase/copy.m

```diff
@@ -0,0 +1,7 @@
+function wout = copy(w)
+% Copy this DnD object or an array of DnD objects
+%
+%   >> wout = copy(w);
+
+    wout = w;
```

Comment:
Indent here is inconsistent with most of the other Horace functions, which are not indented.